src/gql/documents/mutations/account/accountDescriptionUpdate.ts

```diff
@@ -0,0 +1,20 @@
+import { useMutation } from '@urql/vue'
+import { graphql } from '~~/gql/generated'
+
+export const accountDescriptionUpdateMutation = graphql(`
+  mutation AccountDescriptionUpdate(
```

Comment:
```suggestion
  mutation AccountUpdate(
```
or does `AccountPatch` only allow to change the description?

Reply:
Renamed and inlined, you can resolve once checked